mutt_notmuch.c

```diff
@@ -611,13 +502,13 @@ static const char *get_db_filename(struct NmCtxData *data)
   if (!data)
     return NULL;
 
-  db_filename = data->db_filename ? data->db_filename : NmDefaultUri;
+  db_filename = data->db_url.path ? data->db_url.path : NmDefaultUri;
   if (!db_filename)
     db_filename = Folder;
   if (!db_filename)
     return NULL;
   if (strncmp(db_filename, "notmuch://", 10) == 0)
-    db_filename += 10;
+    db_filename += 9;
```

Comment:
I'm not sure about this change, though I couldn't work out what settings would actually get me there.
A url with a path in it would have '///' in it (three slashes), so
`notmuch:///home/user/nm` would become `//home/user/nm`

Reply:
That's a leftover, it should be 10. I remember having changing this to exercice a bug in url_parse().